### ISSUE TITLE
New release v0.8.18

### ISSUE THREAD

**Issue by diesl** [summernote/summernote-rails#89]
Hi,

can you make a new release with latest version of summernote please?

The last release 0.8.12 is already one year old.

Thanks!

**Comment by Bertrand-Bichat** [comment]
> Hi,
> 
> can you make a new release with latest version of summernote please?
> 
> The last release 0.8.12 is already one year old.
> 
> Thanks!

I agree, we can see here : [https://rubygems.org/gems/summernote-rails](https://rubygems.org/gems/summernote-rails)

**Comment by m-z-b** [comment]
Could you push the current version (0.18.17) to rubygems.org? 

The version there is still 0.18.12 which has some annoying bugs related to inserting tables.

Thanks.
Note. I tested the update procedure in MAINTAIN.md and I didn't find any issues with 0.8.18.0.

**Comment by Rabin-Kalikote** [comment]
Please release v0.8.18
@luciuschoi Please help.

**Comment by diesl** [comment]
/ping @luciuschoi:

I have seen that you pushed to rubygems last time (version 0.18.12). Could you please help here, too?
